### ISSUE TITLE
Select single user for admin's Advance Permission ("user/group can manage this folder") doesn't work

### ISSUE THREAD

**Issue by s8321414** [nextcloud/groupfolders#3388]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. Create a group folder
2. Set groups that can access this group folder
3. Check "Advance Permission" box and choose single user (non-admin) in dropdown menu

### Expected behaviour
Chosen user can manage selected group folder.

### Actual behaviour
Chosen user can't manage selected group folder (there are not any group folder admin options in normal group folder UI). But we can set a group that is only for manage this group folder, it works.

### Server configuration

**Operating system:** Rocky Linux 8

**Web server:** Apache 2.4

**Database:** MariaDB 10.6

**PHP version:** 8.3

**Nextcloud version:** (see Nextcloud admin page) 30.0.1 (also in older version)

**Group folders version:** 18.0.3

**Updated from an older Nextcloud/ownCloud or fresh install:** Fresh install (also tested updated for older Nextcloud)

**Where did you install Nextcloud from:** Nextcloud website download (also tested on Docker)

**Are you using external storage, if yes which one:** no

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** no

### Client configuration
**Browser:** Firefox 131.0.3/Google Chrome 130.0.6723.69

**Operating system:** Arch Linux

**Comment by provokateurin** [comment]
Is the user you want to select part of the group you added to the Groupfolder? Otherwise it is expected that no user will show up.

**Comment by s8321414** [comment]
> Is the user you want to select part of the group you added to the Groupfolder? Otherwise it is expected that no user will show up.

Yes, that user is in group selected for group folder so I can select that user in Advance Permission (in admin UI) dropdown menu.

**Comment by provokateurin** [comment]
So the user is not seeing the ACL settings in the sidebar, right?

**Comment by s8321414** [comment]
> So the user is not seeing the ACL settings in the sidebar, right?

Yes, ACL settings don't show for selected user.

**Comment by provokateurin** [comment]
I just tried reproducing it, but for me the sidebar shows up correctly for the assigned user.
Can you please check if there are any errors in the nextcloud, webserver and browser logs?

**Comment by s8321414** [comment]
Ah, for database it is not correct, should be SQLite not MariaDB, sorry.
